.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```

Comment:
Znowu folderu .idea nie powinno tu być. Powinien trafić do .gitignore

Reply:
jest w gitignore - po prostu nie jest usunięty. to różnica